crates/examples/microkit/http-server/pds/virtio-net-driver/Cargo.toml

```diff
@@ -8,14 +8,16 @@ license = "BSD-2-Clause"
 [dependencies]
 log = "0.4.17"
 microkit-http-server-example-virtio-hal-impl = { path = "../../helpers/virtio-hal-impl" }
-microkit-http-server-example-virtio-net-driver-interface-types = { path = "./interface-types" }
+microkit-http-server-example-smoltcp-phy-impl = { path = "../../helpers/smoltcp-phy-impl" }
 sel4 = { path = "../../../../../sel4" }
 sel4-bounce-buffer-allocator = { path = "../../../../../sel4-bounce-buffer-allocator" }
 sel4-externally-shared = { path = "../../../../../sel4-externally-shared", features = ["unstable"] }
+sel4-hal-adapters = { path = "../../../../../sel4-hal-adapters/smoltcp" }
```

Comment:
Should be:

```
sel4-hal-adapters = { path = "../../../../../sel4-hal-adapters" }
```

The `Run other tests` CI job didn't catch this because the Nix build patches these paths, but I believe the `Check source` job would have.